pyproject.toml

```diff
@@ -30,6 +30,7 @@ dependencies = [
 "floris~=4.3",
 "nrel-pysam~=6.0",
 "netCDF4",
+"rainflow==3.2",
```

Comment:
Consider using a compatible release specifier (`"rainflow~=3.2"`) to allow the version to float upwards